uk.co.compucorp.civicrm.hrcontactactionsmenu/css/contactactions.css

```diff
@@ -0,0 +1,67 @@
+.page-civicrm-contact-view #crm-contact-actions-link.active,
```

Comment:
And then we should create scss partials for different things. You can refer to L&A extension.
scss
 - base
    - variables.scss (this should contain all custom variables for this extension)
 - components (all custom components, in this case I can see crm_contact_action_menu__user_info and crm_contact_action_menu__body)
    - contact_action_menu__user_info.scss
    - contact_action_menu__body.scss

Reply:
I agree @deb1990 The gulp file will be minimal as it will be used for compiling ans watching `scss`  partials only. So, I think it wont't bring any complexity.